src/components/SearchBar.tsx

```diff
@@ -18,6 +18,13 @@ export default function SearchBar() {
 
   useDebounce(() => setSearchText(value), 200, [value])
 
+  document.body.addEventListener("keydown", (event) => {
+    if (event.ctrlKey && event.key === "f") {
```

Comment:
This is not so good, as it creates a new listener everytime it rerenders and it does not clean up the previous listener. You need to use a useEffect hook with a cleanup function.